packages/app-project/src/helpers/theme/README.md

```diff
@@ -0,0 +1,5 @@
+# Theme
+
+Creates a Grommet theme by performing a deep merge of the Grommet base theme with the Zooniverse custom Grommet theme.
+
+This was originally done in the `GrommetWrapper` theme, but was ,oved here so it can be reused with `@artsy/fresnel` to extract the breakpoints for responsive sizing.
```

Comment:
```suggestion
This was originally done in the `GrommetWrapper` theme, but was moved here so it can be reused with `@artsy/fresnel` to extract the breakpoints for responsive sizing.
```